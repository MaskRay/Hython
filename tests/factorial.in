def factorial(n):   # comment test
    if n < 2:
        return 1
    return factorial(n-1) * n

print factorial(17)
