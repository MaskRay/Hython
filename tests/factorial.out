355687428096000
